### PR TITLE
Add SELinux Check

### PR DESCRIPTION
**Pull request by q384566678**
1. update godeps and add go-selinux deps
2. add selinux check in validate and runtimetest

Close #386 

This is a rebase of #386 

Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by q384566678** [comment]
@liangchenye  PTAL

**Comment by liangchenye** [comment]
@q384566678 I'll review it today.

File: validation/linux_mount_label/linux_mount_label.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import (
+	"github.com/opencontainers/runtime-tools/validation/util"
+)
+
+func main() {
+	g, err := util.GetDefaultGenerator()
+	if err != nil {
+		util.Fatal(err)
+	}
+	g.SetLinuxMountLabel("system_u:object_r:svirt_sandbox_file_t:s0:c715,c811")
```
Comment (liangchenye) [inline validation/linux_mount_label/linux_mount_label.go]: We need an 'invalid mount label' test case as well.

**Comment by liangchenye** [comment]
LGTM

As commented, We need an 'invalid mount label' test case as well.

**Pull request merged**